cerealize/test.cc

```diff
@@ -166,31 +173,16 @@ TEST(Cerealize, JSONSerializesAndParses) {
   EventAppSuspend b;
   EventAppResume c;
 
-  GenericCerealFileAppender<CerealFormat::JSON>(CurrentTestTempFileName()) << a << b;
-  GenericCerealFileParser<MapsYouEventBase, CerealFormat::JSON> f(CurrentTestTempFileName());
+  CerealFileAppender<CerealFormat::JSON> f1(CurrentTestTempFileName());
+  f1 << a;
+  EXPECT_EQ(1u, f1.EntriesWritten());
+  EXPECT_EQ(169u, f1.BytesWritten());
+  CerealFileAppender<CerealFormat::JSON> f2(CurrentTestTempFileName());
+  f2 << b << c;
+  EXPECT_EQ(2u, f2.EntriesWritten());
+  EXPECT_EQ(340u, f2.BytesWritten());
 
-  std::ostringstream os;
-  while (f.Next([&os](const MapsYouEventBase& e) { e.AppendTo(os) << '\n'; }))
-    ;
-
-  EXPECT_EQ(
-      "Type=EventAppStart, ShortType=\"a\", UID=, UID_Google=, UID_Apple=, UID_Facebook=, foo=foo\n"
-      "Type=EventAppSuspend, ShortType=\"as\", UID=, UID_Google=, UID_Apple=, UID_Facebook=, bar=bar\n",
-      os.str());
-}
-
-TEST(Cerealize, BinaryStreamCanBeAppendedTo) {
-  FileSystem::MkDir(FLAGS_cerealize_test_tmpdir, FileSystem::MkDirParameters::Silent);
-  FileSystem::RmFile(CurrentTestTempFileName(), FileSystem::RmFileParameters::Silent);
-
-  EventAppStart a;
-  EventAppSuspend b;
-  EventAppResume c;
-
-  CerealFileAppender(CurrentTestTempFileName()) << a << b;
-  CerealFileAppender(CurrentTestTempFileName()) << c;
-
-  CerealFileParser<MapsYouEventBase> f(CurrentTestTempFileName());
+  CerealFileParser<MapsYouEventBase, CerealFormat::JSON> f(CurrentTestTempFileName());
```

Comment:
I would love to see `CerealFileFormat::JSON` as the default setting. (Link to the same gist: https://gist.github.com/dkorolev/ce687e33b642e2de0ce3).